docs/installation/working-with-typescript.md

```diff
@@ -0,0 +1,66 @@
+---
+# Scope:
+# * Introduction to TypeScript in CKEditor 5
+# * List and clarify the things that need attention when using TypeScript.
+
+
+category: installation
+order: 29
+modified_at: 2023-03-29
+---
+
+# TypeScript support
+
+CKEditor 5 is built using TypeSript and so has native types definition.
+
+<info-box hint>
+Using TypeScript is just an option so if you don't need its features you can continue using CKEditor 5 in JavaScript.
+</info-box>
+
+## Why to use CKEditor 5 with TypeScript?
```

Comment:
```suggestion
## Reasons to use CKEditor 5 with TypeScript
```

Reply:
Let's leave "Why" for now. The "Reasons" sound like we need to prove that it's worth using TypeScript. I'm a fan of listing reasons, but in this case, it may backfire as the reasons may vary depending on the context of use.